node/Cargo.toml

```diff
@@ -10,7 +10,7 @@ anyhow = "1.0.92"
 async-trait = "0.1.83"
 axum = "0.7.9"
 borsh = { version = "1.5.1", features = ["derive"] }
-cait-sith = { git = "https://github.com/kuksag/cait-sith", rev = "84626e50b3bf9135275f9ae610d05674bcba661f" }
+cait-sith = { git = "https://github.com/kuksag/cait-sith", rev = "e89e3d96900c52f4e1097d4031dd8ee3b1eb60df" }
```

Comment:
Blocked on https://github.com/Near-One/cait-sith/pull/12

Reply:
JFYI: Original line also should've been "github.com/**Near-One**/cait-sith", instead of "github.com/**kuksag**/cait-sith".
It was brought by mistake in https://github.com/Near-One/mpc/pull/303, sorry